presto/inference.py

```diff
@@ -225,7 +226,7 @@ def _get_encodings(self, dl: DataLoader) -> np.ndarray:
             
             try:
                 x, dw, latlons, month, variable_mask = b
-            except:
+            except ValueError:
```

Comment:
Do we have any idea on the overhead required for this try/except in the dataloader? I'm wondering if it would make more sense to have another child class specifically for the DataFrame case?